AU2/plugins/custom_plugins/PolicePlugin.py

```diff
@@ -0,0 +1,228 @@
+import datetime
+import os
+import random
+from typing import List
+
+from AU2 import ROOT_DIR
+from AU2.database.AssassinsDatabase import ASSASSINS_DATABASE
+from AU2.database.EventsDatabase import EVENTS_DATABASE
+from AU2.database.GenericStateDatabase import GENERIC_STATE_DATABASE
+from AU2.database.model import Assassin, Event
+from AU2.html_components import HTMLComponent
+from AU2.html_components.AssassinDependentIntegerEntry import AssassinDependentIntegerEntry
+from AU2.html_components.AssassinDependentSelector import AssassinDependentSelector
```

Comment:
This import doesn't appear needed. Can you check which of these you need and delete those you don't?